ftplugin/markdown.vim

```diff
@@ -14,6 +14,14 @@ if !exists("g:vmt_dont_insert_fence")
     let g:vmt_dont_insert_fence = 0
 endif
 
+if !exists("g:vmt_list_item_char")
+    let g:vmt_list_item_char = '*'
+endif
+
+if !exists("g:vmt_list_indent_text")
+    let g:vmt_list_indent_text = '  '
```

Comment:
This section will change the default behavior.  😄

Reply:
what is the default value? it is not two spaces?

Reply:
No, it isn't.

You can see my original `GetIndentText()` function:

```viml
function! s:GetIndentText()
    if &expandtab
        return repeat(" ", &shiftwidth)
    else
        return "\t"
    endif
endfunction
```

Reply:
ok, now it is : 
```vim
function! s:GetIndentText()
    if !empty(g:vmt_list_indent_text)
        return g:vmt_list_indent_text
    endif
    if &expandtab
        return repeat(" ", &shiftwidth)
    else
        return "\t"
    endif
endfunction
```

so I need to change the default value of `g:vmt_list_indent_text` to a empty string, if user do not set this option, then it keep the default behavior .